### DISCUSSION TITLE
How to customize / design welcome email?

### DISCUSSION THREAD

**Discussion by InDerectif** [requarks/wiki#3409] [Help / Questions]
**Question**
Hi All,

i have already set up wiki.js and i am very happy with the features i have so far. Especially the whitelisiting function for the registered users is great.

My question refers to the email which is sent for verification: Is it possible to style it? Here I'm especially interested in the image (https://static.requarks.io/email/email-cover-book.jpg), which is included initially.

I would be very happy about workarounds and solution suggestions.

Thanks and greetings
Max

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.4.107
Database engine: postgres 11.8

**Comment by InDerectif** [comment]
Hi All,

not the best option but we just changed the email template manual: `server/templates/account-verify.html`

Important: You need to restart the wiki to activate your changes.

**Comment by roshanjonah** [comment]
Do you guys know how to change the default font from Roboto to something like Lato? @InDerectif - Much appreciated :)
